docarray/typing/url/mesh_url.py

```diff
@@ -0,0 +1,83 @@
+from typing import TYPE_CHECKING, Any, Tuple, Type, TypeVar, Union
+
+import numpy as np
+
+from docarray.typing.url.any_url import AnyUrl
+from docarray.typing.url.helper_3d_data import MESH_FILE_FORMATS, _load_trimesh_instance
+
+if TYPE_CHECKING:
+    from pydantic import BaseConfig
+    from pydantic.fields import ModelField
+
+    from docarray.proto import NodeProto
+
+T = TypeVar('T', bound='MeshUrl')
+
+
+class MeshUrl(AnyUrl):
```

Comment:
lets have an abstract 3dUrl class that implement the `_load_trimesh_instace` rather than having it in the helper